### DISCUSSION TITLE
nvidia is deprecated

### DISCUSSION THREAD

**Discussion by bwanaaa** [jef/streetmerchant#1327] [Help] [answered]
starting bot produces

[11:23:18 AM] warn :: nvidia is deprecated in favor of bestbuy

what does this mean?

**Comment by jef** [comment, marked as answer]
This means if you're using the store `nvidia` and in the US, it no longer works. Instead, use BestBuy, as Nvidia has publicly stated that's where they will be selling cards.

**Reply by bwanaaa** [reply]
thank you. Where can I make this change as well as specify other sites? It's not obvious in the .env file

**Reply by phtp** [reply]
Don't add Nvidia as an option.

**Reply by jef** [reply]
> thank you. Where can I make this change as well as specify other sites? It's not obvious in the .env file

It is in the `.env`.

`STORES=amazon`

https://jef.codes/streetmerchant/getting-started/
